aces/Loris
# [DQT] DQT redesign with existing dataquery

## Brief summary of changes

I made #6018 work alongside the existing dataquery.
![exx](https://user-images.githubusercontent.com/16293415/87468463-adaaf180-c5e7-11ea-8fcc-694c1a9b0e92.jpg)

#### Testing instructions (if applicable)

1. Test the "Data Query Tool (Beta)" module.
2. You may need to refresh the database. `Loris/tools/raisinbread_refresh.php`
3. Confirm the functionality still works of the dqt module redesign.

**Note:** Any new bugs found that exist in the non-redesigned dataquery should be separate from this PR.
